modules/telestion-application/src/main/java/de/wuespace/telestion/application/launcher/Launcher.java

```diff
@@ -0,0 +1,83 @@
+package de.wuespace.telestion.application.launcher;
+
+import io.vertx.core.Future;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.lang.Thread.sleep;
+
+/**
+ * A {@link Launcher} is the first part of the launch process in Telestion.
+ * <p>
+ * It starts other processes like {@link de.wuespace.telestion.application.loader.Loader Loaders}
```

Comment:
might be part of on of the other stacked PRs but I cannot find this class :D

Reply:
Jep, I've written everything in one run and splitted it into smaller pieces for a better review experience. This JavaDoc error disappears once we merge the stack from above. :wink: